app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/UIHelper.cs

```diff
@@ -130,13 +130,45 @@ public static IEnumerable<Face> CalculateFaceRectangleForRendering(IList<Detecte
 
             foreach (var face in faces)
             {
+                var Left = (int)((face.FaceRectangle.Left * scale) + uiXOffset);
+                var Top = (int)((face.FaceRectangle.Top * scale) + uiYOffset);
+
+                // FaceAngle use for roate face Rect, default is 0(no rotate).
+                double FaceAngle = 0;
+
+                // If there has headpose attributes, will re-calculate the left/top(X/Y) position.
+                if (face.FaceAttributes?.HeadPose != null)
+                {
+                    // Headpose's roll value act directly on face angle. 
```

Comment:
`Headpose's roll value acts directly as the face angle.`

Reply:
Done.